src/components/Upload.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+
+const Upload = ({ setCurrentScript, titles, setTitles }) => {
+  const MAXFILESIZE = 50 * 1024 * 1024; // 50MB
```

Comment:
One thing that can be helpful for number constants like this is to have a suffix denoting the unit, like `MAX_FILESIZE_BYTES`